cassandra_slo/mininet/topology_slo.py

```diff
@@ -50,9 +56,23 @@ def __init__(self):
         self.startCassandra('h1')
         self.startCassandra('h2')
 
+    # TESTING CORE PINNING (returns number of cores)
+	numOfCores = numCores()
+	print 'The number of cores is: ', numOfCores
+
+    # Assign cores to the hosts
+	self.getHost('h1').config(cores=1)	
```

Comment:
@ShubhankerS is anything missing from here? Hannes says that he added the things from the server. Please make sure that nothing is missing.

Ping @PethrusG also.

Reply:
This has been removed.